components/filter-components/MapFilters.vue

```diff
@@ -49,6 +49,19 @@
                   Accused Witches
                 </p>
               </div>
+              <div  class="ml-3 flex mt-1y items-center pb-2" v-if="!iconsConstant">
+                <p class="text-base witchy-text mr-2">
+                    Colour of Map Icons Indicating
```

Comment:
only comment from me is to change the casing here to be _Colour of map icons indicating_ instead. Thanks Ruby

Reply:
just done